utils/ttyd/files/ttyd.init

```diff
@@ -0,0 +1,98 @@
+#!/bin/sh /etc/rc.common
+
+START=99
+STOP=50
+USE_PROCD=1
+
+NAME=ttyd
+PROG=/usr/bin/$NAME
+
+validate_section_ttyd()
+{
+	uci_load_validate ttyd ttyd "$1" "$2" \
+		'enable:bool:1' \
```

Comment:
hardcoded to `1`?

Reply:
No, it means that it defaults to `1`.

Reply:
thanks @jow- .

I think we should disabled it by default, and put all default values to the config file (`ttyd.config`). @ysc3839

Reply:
@tsl0922 dropbear is enabled by default.
https://github.com/openwrt/openwrt/blob/524810ce6ddff85a79af9ed979452bb8a8ed1b6d/package/network/services/dropbear/files/dropbear.init#L119